spec/appliance_console/key_configuration_spec.rb

```diff
@@ -80,21 +82,34 @@
           subject.force = true
           v2_exists(true) # before downloaded
           v2_exists(true) # after downloaded
-          expect(FileUtils).to receive(:rm).with(/v2_key/).and_return(["v2_key"])
           scp = double('scp')
           expect(scp).to receive(:download!).with(subject.key_path, /v2_key/).and_return(:result)
           expect(Net::SCP).to receive(:start).with(host, "root", :password => password).and_yield(scp).and_return(true)
+          expect(FileUtils).to receive(:mv).with(/v2_key\.tmp/, /v2_key$/).and_return(true)
           expect(subject.activate).to be_truthy
         end
 
         it "fails if key exists (no force)" do
           expect($stderr).to receive(:puts).at_least(2).times
           subject.force = false
           v2_exists(true)
-          expect(FileUtils).not_to receive(:rm)
+          expect(FileUtils).not_to receive(:mv)
           expect(Net::SCP).not_to receive(:start)
           expect(subject.activate).to be_falsey
         end
+
```

Comment:
@ailisp my mistake, you have removed back-up in latest edition, disregard last comment